suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -99,9 +99,11 @@ export const prepareTransactionsReducer = createReducerWithExtraDeps(
             .addCase(transactionsActions.removeTransaction, (state, { payload }) => {
                 const { account, txs } = payload;
                 const transactions = state.transactions[account.key];
-                state.transactions[account.key] = transactions?.filter(
-                    tx => !txs.some(t => t.txid === tx?.txid),
-                );
+                if (transactions) {
```

Comment:
I see there is a test file for this, it may be worth to add a test-case for this fix :pray: 

`suite-common/wallet-core/src/transactions/__tests__/transactionsReducer.test.ts`

Reply:
https://github.com/trezor/trezor-suite/pull/12951/commits/44e26b5c44b549c7632e81d42ef42c045bc725ce